### ISSUE TITLE
 Increased anti-viral antibody level

### ISSUE THREAD

**Issue by jpgourdine** [obophenotype/human-phenotype-ontology#4869]
**Preferred term label:**
 Increased anti-viral antibody level

**Synonyms**
Increased level of anti-viral antibody

**Definition (free text, please give PubMed ID)**
An increase of the level of antibody against virus.

**Parent term (use hpo.jax.org/app)**

Increased antibody level in blood (HP:0010702)

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**
0000-0001-9969-8610

**Comments**
Should we create a subclass with increased anti-HIV, Hepatitis A, B, C. etc. or just stay at the high level  "Increased anti-viral antibody level" for the LOINC ?

**Comment by pnrobinson** [comment]
I am not sure that the overall level of antiviral antibody is measurably altered -- arent we actually talking about specific antibodies? And do not measurements typically involved IgG and IgM?

**Comment by LCCarmody** [comment]
It sounds like there are Anti-HIV/Hep antibody tests in LOINC...and he is wondering if we should do each individual virus or just create one higher level, parent term so all terms would be curated to.  I think?  Anyways, I vote for a parent term of Increased anti-viral antibody, with children Increased anti-HIV antibody, and Increased Hep A/B/C because they definitely impact treatment. For other viruses, I am not sure if we need separate for flu or others.

**Comment by nicolevasilevsky** [comment]
There are these terms in LOINC:

- Hepatitis A virus Ab
- Hepatitis A virus Ab.IgM
- Hepatitis B virus core Ab
- Hepatitis B virus core Ab.IgM
- Hepatitis B virus surface Ab
- Hepatitis C virus Ab

**Comment by pnrobinson** [comment]
let's definitely add terms for these (note I have the lock right now and am working on other terms)
